### PR TITLE
Features

### LINKED ISSUE #42: Remove the index export pattern
Related comments:
https://github.com/koh1project/apprenticeship_to_master/pull/40#discussion_r990587518
https://github.com/koh1project/apprenticeship_to_master/pull/40#discussion_r990587695

### PR DESCRIPTION
**Pull request by koh1project**
## What I have done
- Published one article
- Set lint rules
- Identify one type of error
- Modify index export pattern

## What I had already known
- OOP approach
- ESLint would fix import order

## What I have acquired this week
- The apollo-server-testing library was deprecated and I did not check it, resulting in causing one typing error and using ts-ignore. I should have checked the library information in the official document. 
- The experience to publish an article. It took a long time and needed effort, and also realized me what I did not clearly knew the target information.
- Understood why the index export pattern seems like an anti-pattern.

closes #42 #41 #43

### CODE REVIEW

File: portfolio_gallery/backend/src/schema/resolvers-types.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
+import { GraphQLResolveInfo } from 'graphql';
```
Comment (kei95) [inline portfolio_gallery/backend/src/schema/resolvers-types.ts:1]: [nitpick] - it's just a common practice that has nothing to do with performance or anything, it's common the leave a space between import and others line below;
```suggestion
import { GraphQLResolveInfo } from 'graphql';

export type Maybe<T> = T | null;
export type InputMaybe<T> = Maybe<T>;
```

**Review by kei95 — APPROVED** [review]
@koh1project Good job on the articles! If you haven't done yet, it's a good idea to run prettier in the project coz I found some inconsistency of lines in `resolvers-types.ts`. Also, let's get this PR merged before it gets too big

File: portfolio_gallery/backend/src/schema/resolvers-types.ts (hunk 1 above)
Comment (koh1project) [reply portfolio_gallery/backend/src/schema/resolvers-types.ts:1]: Thank you Kei, OK I think it does not take a long time. I will try to set format rules and set it on commit!
#45

**Comment by kei95** [comment]
@koh1project I'm confused by the commit history and this PR. Would you create a different PR for your changes? I don't think it's a good idea to reopen closed PR and add changes to it. You can even create different branches for each change as it's not realistic to review thousands of lines at once. Make sure not to open a big PR since that would increase the chance to slip errors in. It's pretty important to keep your PR easy to review because of it

**Comment by koh1project** [comment]
@kei95 I am sincerely sorry for making confusion. I made 2 mistakes, forgot to merge on the browser, and used the same branch name, resulting in this PR page unintentionally containing additional changes including the newest PR's changes. I will make sure such things never happen again.

**Comment by kei95** [comment]
> 
@kei95 I am sincerely sorry for making confusion. I made 2 mistakes, forgot to merge on the browser, and used the same branch name, resulting in this PR page unintentionally containing additional changes including the newest PR's changes. I will make sure such things never happen again.

No need to apologize!

**Pull request merged**